src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/client/PrisonApiClient.kt

```diff
@@ -46,24 +42,6 @@ class PrisonApiClient(
       }
       .block(apiTimeout)
   }
-
-  fun getPrisonerDetails(offenderNo: String): PrisonerDetailsDto? {
-    LOG.debug("Entered getPrisonerDetails $offenderNo")
-    return webClient.get()
-      .uri("/api/prisoners/$offenderNo/full-status")
-      .retrieve()
-      .bodyToMono(TYPE_FOR_PRISONER_DETAILS)
-      .onErrorResume { e ->
-        if (!isNotFoundError(e)) {
-          LOG.error("getPrisonerDetails Failed get request /api/prisoners/$offenderNo/full-status")
-          Mono.error(e)
-        } else {
-          LOG.debug("getPrisonerDetails Not Found get request /api/prisoners/$offenderNo/full-status")
-          return@onErrorResume Mono.justOrEmpty(null)
-        }
-      }
-      .block(apiTimeout)
-  }
```

Comment:
not used any more I assume

Reply:
yes not needed anymore. there were 2 calls being made one to Prison API and another to Prisoner offender search to get similar details. As this data can be retrieved from the call to  Prisoner offender search the call to Prison API has been removed.